edb/lib/std/25-numoperators.edgeql

```diff
@@ -2680,3 +2680,47 @@ CREATE CAST FROM std::decimal TO std::str {
     SET volatility := 'Immutable';
     USING SQL CAST;
 };
+
+
+## Bytes casts.
```

Comment:
I think my inclination is to take the functions but omit adding the casts.

Do you have an argument in favor of doing both?

Reply:
My only argument for adding the casts would be consistency. So if, aside from `json`, conversion is solely done via functions it should be the same here.